OWNERS_ALIASES

```diff
@@ -31,39 +30,29 @@ aliases:
   - enxebre
 
   # -----------------------------------------------------------
-  # OWNER_ALIASES for docs/book
+  # OWNER_ALIASES for bootstrap/kubeadm
   # -----------------------------------------------------------
 
-  # folks who can review and LGTM any PRs under docs/book
-  cluster-api-book-reviewers:
-  - moshloop
-  - randomvariable
```

Comment:
Remove the specific reviewers for the book here given that it's probably all folks should do, 

cc @randomvariable @moshloop